eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -0,0 +1,43 @@
+param(
+  $RepoOwner,
+  $RepoName,
+  $BranchPrefix,
+  $WorkingDirectory,
+  $AuthToken
+)
+
+. "${PSScriptRoot}\common.ps1"
+
+pushd $WorkingDirectory
+git clone https://github.com/$RepoOwner/$RepoName.git
+pushd $RepoName
+$syncBranches = git branch -r --list origin/$BranchPrefix* | % { $_ -replace "origin/", "" }
+
+LogDebug "Operating on Repo [ $RepoName ]"
+foreach ($branch in $syncBranches)
+{
+  try {
+    $branchName = $branch.Trim()
```

Comment:
If it is interesting to trim then I would do it above when producing the list of branches.